fraud-analytics/linux_inference_endpoint.py

```diff
@@ -0,0 +1,160 @@
+from flask import Flask, request, jsonify
```

Comment:
why not move this file along with containerfile to `app` directory?

Reply:
Ok things got messed up when I rebased with main. will fix it.